website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+
+If your app or library relies on `PropTypes`, we highly recommend migrating to a type system like TypeScript. **If you have trouble identifying the source of a deleted `PropTypes` usage, please [file an issue](https://github.com/facebook/react-native/issues) so we can help track it down with you.**
+
+## Deprecated APIs in `PushNotificationIOS`
+
+:::warning
+The [PushNotificationIOS](https://reactnative.dev/docs/pushnotificationios) library is marked as deprecated. The changes in this release are focused on removing references to deprecated iOS APIs. In a future release, this library will be moved out of react-native core and converged with the community package, [@react-native-community/push-notification-ios](https://github.com/react-native-push-notification/ios).
+:::
+
+[PushNotificationIOS](https://reactnative.dev/docs/pushnotificationios) has been migrated onto Apple’s [User Notification](https://developer.apple.com/documentation/usernotifications?language=objc) framework and exposes new APIs for scheduling and handling notifications. If you are still relying on PushNotificationIOS, you’ll need to migrate over before the next release (0.75) when the deprecated APIs will be removed.
+
+### API Changes
+
+The `didRegisterUserNotificationSettings:` callback on `RCTPushNotificationManager` was a no-op and has been deleted.
+
+The following callbacks on `RCTPushNotificationManager` have been deprecated and will be removed in 0.75:
+
+```objectivec
++ (void)didReceiveLocalNotification:(UILocalNotification *)notification;
++ (void)didReceiveRemoteNotification:(NSDictionary *)notification;
+```
+
+In order to retrieve the notification which launched the app using `getInitialNotification()`, you’ll now need to explicitly set the `initialNotification` on `RCTPushNotificationManager`:
+
+```objectivec
+[RCTPushNotificationManager setInitialNotification:response.notification];
+```
+
+On the JS side, properties on `Notification` have changed. `alertAction` and `repeatInterval` are now deprecated and will be removed in 0.75:
+
+```js
+type Notification = {|
+ ...
+ +fireDate?: ?number,
+ /** NEW. Seconds from now to display the notification. */
+ +fireIntervalSeconds?: ?number,
+ /**
+  * CHANGED. Used only for scheduling notifications. Will be null when
+  * retrieving notifications using `getScheduledLocalNotifications` or
+  * `getDeliveredNotifications`.
+  */
+ +soundName?: ?string,
+ /** DEPRECATED. This was used for iOS's legacy UILocalNotification. */
+ +alertAction?: ?string,
+ /** DEPRECATED. Use `fireDate` or `fireIntervalSeconds` instead. */
+ +repeatInterval?: ?string,
+|};
+
+```
+
+Finally, the `handler` parameter on `PushNotificationIOS.removeEventListener` is unused and has been removed.
+
+### How to Migrate
```

Comment:
This feels it should be moved to a dedicated post or a page somewhere else (discussions-and-proposals)?

Reply:
This or hidden under `<details>` block